context/context.go

```diff
@@ -0,0 +1,26 @@
+package context
+
+import (
+	"context"
+
+	"github.com/hellofresh/stats-go/client"
+)
+
+type statsKeyType int
+
+const statsKey statsKeyType = iota
+
+// New returns a context that has a stats Client
+func New(ctx context.Context, client client.Client) context.Context {
+	return context.WithValue(ctx, statsKey, client)
+}
+
+// WithContext returns a stats Client with as much context as possible
+func WithContext(ctx context.Context) client.Client {
+	ctxStats, ok := ctx.Value(statsKey).(client.Client)
+	if !ok {
+		return client.NewNoop(true)
```

Comment:
let's remove `unicode` parameter from `noop` client as it does not make any sense here

Reply:
so should I set it to false?